channels.go

```diff
@@ -0,0 +1,134 @@
+package telebot
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// ChatInfo struct is the main chat information from telegram
+type ChatInfo struct {
+	ID       int64  `json:"id"`
+	Title    string `json:"title"`
+	Username string `json:"username"`
+	Type     string `json:"type"`
+}
+
+// For each member of a chat , we have some information like usertype (admin , etc..) 
+// and the main information about user like id and etc...
```

Comment:
Mimicking the official docs, a better description may be "ChatMember contains information about one member of a chat."